english/make-friends/where_am_i/where_am_i.md

```diff
@@ -0,0 +1,31 @@
+You realise you need as much information as you need to get out of this situation. 
+
+And the only thing you can ask about this seems to be Marshmellow man. You decide to befriend him.
+
+"Hello, Marshmellow man..er.. uh..Do you have a name?" , I asked, not really expecting any answer and wondering 
+
+whether I have gone crazy to be talking with a marshmellow.
+
+"My name is Marshy", mellowed a sweet sounding sing songy voice. "And I am a woman", it said. 
+
+I was a little embarrassed and surprised to hear back a reply. "Where am I?", I asked. 
+
+"This planet is called UR58F3 and about 10 light years away from Planet Earth. 
+
+Your Earth started to literally disintegrate into pieces. So our leaders transported all the remaining humans here."
+
+I was a little relieved. But my mind was beginning to swarm with all sorts of questions. 
+
+"So, how long have I been in hyper-sleep?", I enquired.
+
+"About 10 months", Marshy replied. "you were one of the last group to be transported here. 
+
+We always leave humans in sleep to assess their physical condition and to tune them up to 
+
+suit to this Planet's environment', she intoned.
+
+I couldn't wait to go out and explore the new Planet. But I was feeling very tired and felt like didn't have any energy left.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.